samza-sql/src/main/java/org/apache/samza/sql/translator/JoinTranslator.java

```diff
@@ -154,6 +161,7 @@ void translate(final LogicalJoin join, final TranslatorContext translatorContext
     // the names from the stream as the lookup needs to be done based on what is stored in the local table.
     return
         inputStream
+            .map(inputMetricsMF)
```

Comment:
For local table, we probably need more metrics. This is the stream side of the metrics. We also want to have some metrics for the table where the source stream is repartitioned and sent to table. We anyways do not support it for now. Could you file a jira for it ?

Reply:
created JIRA: SAMZA-2029